src/server/game/Battlegrounds/ArenaTeam.h

```diff
@@ -173,10 +173,10 @@ class TC_GAME_API ArenaTeam
         int32  GetRatingMod(uint32 ownRating, uint32 opponentRating, bool won);
         float  GetChanceAgainst(uint32 ownRating, uint32 opponentRating);
         int32  WonAgainst(uint32 Own_MMRating, uint32 Opponent_MMRating, int32& rating_change);
```

Comment:
I would fix "int32  " and "float  " and whatever "return_type  " is there too while at it